TestResultsQualityGate/task.json

```diff
@@ -18,15 +18,45 @@
   "minimumAgentVersion": "2.209.0",
   "inputs": [
     {
-      "name": "placeholder",
-      "type": "string",
-      "label": "placeholder",
-      "defaultValue": "placeholder",
+      "name": "type",
+      "type": "picklist",
+      "label": "Type",
+      "defaultValue": "Total Passed",
       "required": true,
-      "helpMarkDown": "Type of issues to be evaluated for the threshold."
+      "helpMarkDown": "Type of test results to be evaluated for the threshold.",
+      "options": {
+        "Total Passed": "Total passed tests",
```

Comment:
The option key contains the space char, it is used by ```instanceNameFormat```. 
![image](https://github.com/parasoft/parasoft-findings-vsts/assets/109122571/f28578e4-aa78-4b09-8d54-d85292af93b6)

But I think it would feel quite strange if used in a YAML file, like:
![image](https://github.com/parasoft/parasoft-findings-vsts/assets/109122571/5a88049e-0b3c-45f3-a122-e3a748363992)

@yshi-parasoft  How do you think bout this? Is it ok to use the vaule with space char in yaml?

Reply:
I agreed. It's better not to include spaces. Actually, I prefer camelCase for the option type. Seems we are not using camelCase for existing option keys.